app/kumactl/cmd/apply/testdata/apply-multiple-resource-same-type.yaml

```diff
@@ -0,0 +1,33 @@
+name: sample1
```

Comment:
please add here `---` as first line just to double check that we are supporting this. I've seen many examples of K8S multi resource YAMLs that starts with `---`